src/elements/emby-checkbox/emby-checkbox.js

```diff
@@ -10,7 +10,7 @@ import 'webcomponents';
     function onKeyDown(e) {
         // Don't submit form on enter
         // Real (non-emulator) Tizen does nothing on Space
-        if (e.keyCode === 13 || e.keyCode === 32) {
+        if ((e.keyCode === 13 || e.keyCode === 32) && !browser.firefox) {
```

Comment:
In this case, you will probably submit form on Firefox.
Looks like Firefox fires `click` even after `preventDefault`.
We can ignore `Space` for Firefox, but  we can also handle `Space` only for Tizen.
```suggestion
        if (e.keyCode === 13 || (e.keyCode === 32 && browser.tizen)) {
```